src/pages/editor/InstitutionSupport.tsx

```diff
@@ -0,0 +1,81 @@
+import LinkIcon from '@mui/icons-material/Link';
+import { LoadingButton } from '@mui/lab';
+import { InputAdornment, TextField, Typography } from '@mui/material';
+import { useMutation } from '@tanstack/react-query';
+import { Field, FieldProps, Form, Formik, FormikProps } from 'formik';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import { updateCustomerInstitution } from '../../api/customerInstitutionsApi';
+import { PageSpinner } from '../../components/PageSpinner';
+import { setCustomer } from '../../redux/customerReducer';
+import { setNotification } from '../../redux/notificationSlice';
+import { RootState } from '../../redux/store';
+import { CustomerInstitution } from '../../types/customerInstitution.types';
+import { dataTestId } from '../../utils/dataTestIds';
+
+export const InstitutionSupport = () => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const customer = useSelector((store: RootState) => store.customer);
+
+  const customerMutation = useMutation({
+    mutationFn: (values: CustomerInstitution) =>
+      updateCustomerInstitution({ ...values, serviceCenterUri: values.serviceCenterUri }),
+    onSuccess: (response) => {
+      dispatch(setCustomer(response.data));
+      dispatch(setNotification({ message: t('feedback.success.update_customer'), variant: 'success' }));
+    },
+    onError: () => dispatch(setNotification({ message: t('feedback.error.update_customer'), variant: 'error' })),
+  });
+
+  return (
+    <>
+      {!customer ? (
+        <PageSpinner />
+      ) : (
+        <Formik initialValues={customer} onSubmit={async (values) => customerMutation.mutate(values)}>
+          {({ values, isSubmitting, setFieldValue }: FormikProps<CustomerInstitution>) => (
+            <Form style={{ display: 'flex', flexDirection: 'column', height: '100%' }}>
+              <Typography variant="h2">{t('editor.institution.institution_support')}</Typography>
+              <Typography>{t('editor.institution.institution_support_description')}</Typography>
```

Comment:
ble veldig trangt? 
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/8be979d8-1fd5-4565-a358-270dc5cc4159)

```suggestion
              <Typography variant="h2" gutterBottom>{t('editor.institution.institution_support')}</Typography>
              <Typography paragraph>{t('editor.institution.institution_support_description')}</Typography>
```

Reply:
kunne godt lagt til paragraph for spacing over inputfeltet også, men your call :)
https://sikt.no/tjenester/nasjonalt-vitenarkiv-nva/hjelpeside-nva